app/services/rapid_ready_for_decision/hypertension_observation_data.rb

```diff
@@ -30,12 +31,13 @@ def filtered_entries(bp_readings)
     end
 
     def transform_entry(raw_entry)
-      entry = raw_entry['resource'].slice('issued', 'component', 'performer')
+      entry = raw_entry['resource'].slice('issued', 'component', 'performer', 'effectiveDateTime')
       practitioner_hash = get_display_hash_from_performer('Practitioner', entry)
       organization_hash = get_display_hash_from_performer('Organization', entry)
       bp_hash = get_bp_readings_from_entry(entry)
 
-      { issued: entry['issued'] }.merge(practitioner_hash, organization_hash, bp_hash)
+      { issued: entry['issued'], effectiveDateTime: entry['effectiveDateTime'] }.merge(practitioner_hash,
+                                                                                       organization_hash, bp_hash)
```

Comment:
Maybe this could be refactored to be configurable through a constant if more data is needed:

```ruby
KEYS = [''issued', 'component', 'performer', 'effectiveDateTime']
...
entry = raw_entry['resource'].slice(KEYS)
```

Then we could tidy up this object literal and merge...